src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
-import './App.css';
+import './styles/App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] For this milestone it is required that you use class-based components as will have a chance to refactor them later. Kindly have a look at the screenshot below from the project requirements and make this a class-based component.
![Screenshot from 2022-02-09 13-33-29](https://user-images.githubusercontent.com/20245164/153181224-90666117-d328-4112-a87b-59debf9f633a.png)